projects/1.18/assets/gobber/gobber2/lang/zh_cn.json

```
@@ -40,81 +44,91 @@
   "block.gobber2.gobber2_block_end": "末影戈伯方块",
   "item.gobber2.gobber2_rod_end": "末影戈伯棒",
   "item.gobber2.gobber2_foo_end": "末影戈伯煤炭",
+
   "item.gobber2.dragon_star": "龙之星",
   "item.gobber2.dragon_elytra": "龙之鞘翅",
-  "item.gobber2.dragon_elytra.tip": "提醒：并不是鞘翅，仅用于合成",
-  "item.gobber2.gobber2_excavator.line1": "这锹能破坏3x3的区域",
+  "item.gobber2.dragon_elytra.tip":"注意：不是鞘翅，仅用于合成",
+
+  "item.gobber2.gobber2_excavator.line1": "这铲子能破坏3x3的区域",
   "item.gobber2.gobber2_hammer.line1": "这锤子可以破坏3x3的区域",
-  "item.gobber2.gobber2_paxel.line1": "结合了镐，斧头和锹",
+  "item.gobber2.gobber2_paxel.line1": "结合了镐，斧头和铲子",
   "item.gobber2.gobber2_tree_axe.line1": "会砍伐整棵树",
   "item.gobber2.gobber2_end.unbreakable": "不毁工具",
+
   "item.gobber2.gobber2_sword": "戈伯剑",
   "item.gobber2.gobber2_pickaxe": "戈伯镐",
   "item.gobber2.gobber2_hammer": "戈伯锤",
-  "item.gobber2.gobber2_excavator": "戈伯锹",
+  "item.gobber2.gobber2_excavator": "戈伯铲",
   "item.gobber2.gobber2_paxel": "戈伯多功能工具",
   "item.gobber2.gobber2_axe": "戈伯斧",
-  "item.gobber2.gobber2_tree_axe": "戈伯树斧",
+  "item.gobber2.gobber2_tree_axe": "戈伯伐木斧",
   "item.gobber2.gobber2_hoe": "戈伯锄",
   "item.gobber2.gobber2_shovel": "戈伯锹",
   "item.gobber2.gobber2_bow": "戈伯弓",
-  "item.gobber2.gobber2_sword_nether": "下界剑",
-  "item.gobber2.gobber2_sword_nether.line1": "可以更好的获取凋灵骷髅头颅",
-  "item.gobber2.gobber2_pickaxe_nether": "下界镐",
-  "item.gobber2.gobber2_hammer_nether": "下界锤",
-  "item.gobber2.gobber2_excavator_nether": "下界锹",
-  "item.gobber2.gobber2_paxel_nether": "下界多功能工具",
-  "item.gobber2.gobber2_axe_nether": "下界斧",
-  "item.gobber2.gobber2_tree_axe_nether": "下界树斧",
-  "item.gobber2.gobber2_hoe_nether": "下界锄",
-  "item.gobber2.gobber2_shovel_nether": "下界锹",
-  "item.gobber2.gobber2_bow_nether": "下界弓",
-  "item.gobber2.gobber2_sword_end": "末影剑",
+
+  "item.gobber2.gobber2_sword_nether": "下界戈伯剑",
+  "item.gobber2.gobber2_sword_nether.line1": "更好获取凋灵骷髅的头颅",
+  "item.gobber2.gobber2_pickaxe_nether": "下界戈伯镐",
+  "item.gobber2.gobber2_hammer_nether": "下界戈伯锤",
+  "item.gobber2.gobber2_excavator_nether": "下界铲",
+  "item.gobber2.gobber2_paxel_nether": "下界戈伯多功能工具",
+  "item.gobber2.gobber2_axe_nether": "下界戈伯斧",
+  "item.gobber2.gobber2_tree_axe_nether": "下界戈伯伐木斧",
+  "item.gobber2.gobber2_hoe_nether": "下界戈伯锄",
+  "item.gobber2.gobber2_shovel_nether": "下界戈伯锹",
+  "item.gobber2.gobber2_bow_nether": "下界戈伯弓",
+
+  "item.gobber2.gobber2_sword_end": "末影戈伯剑",
   "item.gobber2.gobber2_sword_sniper": "狙击手之剑",
-  "item.gobber2.gobber2_sword_sniper.line1": "一把具有特殊能力且不会损坏的剑",
-  "item.gobber2.gobber2_sword_sniper.line2": "右击以发射箭",
-  "item.gobber2.gobber2_sword_sniper.line3": "Shift+右击以切换开/关",
-  "item.gobber2.gobber2_sword_sniper.line4": "狙击能力激活：%s",
+  "item.gobber2.gobber2_sword_sniper.line1": "具有特殊能力的无法破坏之剑",
+  "item.gobber2.gobber2_sword_sniper.line2": "右键以发射箭",
+  "item.gobber2.gobber2_sword_sniper.line3": "Shift + 右键单击以切换开/关",

```

```suggestion
  "item.gobber2.gobber2_sword_sniper.line3": "Shift + 右击以切换开/关",
```
全文要不然`右键单击以`、要不然`右击以`，请全文通查。